src/modules.d.ts

```diff
@@ -39,7 +39,7 @@ declare module 'electron' {
     }
 
     export namespace net {
-        export function request(url: string): any;
+        export function request(url: string|Object): any;
```

Comment:
Did the backing electron code change? Do we use it somewhere other than [here](https://github.com/HadoukenIO/core/pull/423/files#diff-f2ab184e212da874890dc8d44365e5bcR199)

Reply:
where do we pass in an obj?